drivers/wdm/audio/backpln/portcls/purecall.cpp

```diff
@@ -8,20 +8,12 @@
 
 #include "private.hpp"
 
-#ifndef YDEBUG
-#define NDEBUG
-#endif
-
-#include <debug.h>
-
 extern "C" {
-  void
+  VOID
 	  __cxa_pure_virtual()
```

Comment:
```suggestion
void __cxa_pure_virtual()
```
this is some CRT stuff.

Reply:
Reverted.
I'll have a look separately.